common/src/main/kotlin/org/bibletranslationtools/maui/common/io/IReader.kt

```diff
@@ -2,6 +2,6 @@ package org.bibletranslationtools.maui.common.io
 
 import io.reactivex.Single
 
-interface IBooksReader {
+interface IReader {
```

Comment:
I would recommend a different name for this interface. Reader sounds a bit too general and also used by java I/O APIs. "IResourceReader", "ISchemaParser" or "IResourceParser" maybe a better name for it